resources/todomvc/architecture-examples/angular/README.md

```diff
@@ -1,11 +1,27 @@
-# Speedometer 2.1: Angular 4 TodoMVC example
+# Angular
 
-## Test locally
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.2.2.
 
-1. `npm install`
-2. `npm run start-prod`
-3. Navigate to `http://localhost:4200/`
+## Development server
 
-## Build (do this before committing!)
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
```

Comment:
What do you think of refering to the npm scripts rather than `ng`?

Reply:
yeah - the whole readme was generated by the angular cli - it's temp for now